### PR TITLE
move soda/mango and soda/pocket to magic_force/mf17 and magic_force/mf34

### PR DESCRIPTION
**Pull request by gezhaoyou**
## Description

move soda/mango and soda/pocket to magic_force/mf17 and magic_force/mf34
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/magic_force/mf17/readme.md
Comment (zvecr) [inline keyboards/magic_force/mf17/readme.md:7]:
```suggestion

```

**Comment by drashna** [comment]
> respective entries in `data/mappings/keyboard_aliases.hjson` are missing

This ^.

Please add an entry so that the old names are mapped to the new names.

**Comment by gezhaoyou** [comment]
> > respective entries in `data/mappings/keyboard_aliases.hjson` are missing
> 
> This ^.
> 
> Please add an entry so that the old names are mapped to the new names.

I don't know why PR Lint error : 
 Keyboard alias keychron/q1 has a target that doesn't exist: keychron/q1v1
☒ Keyboard alias keychron/q4 has a target that doesn't exist: keychron/q4/ansi

my keyboard was magic_force/xxx

**Comment by drashna** [comment]
The lint error is unrelated to your changes, and can be ignored.

**Review by drashna — APPROVED** [review]

**Comment by gezhaoyou** [comment]
Can someone merge the code, or do I need to resubmit the PR?

**Comment by drashna** [comment]
> Can someone merge the code, or do I need to resubmit the PR?

Generally, we want 2x green approvals before merging. 

That said resubmitting the PRs is frowned upon

**Review by fauxpark — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**
